chemprop/cli/train.py

```diff
@@ -118,7 +132,7 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         "--frzn-ffn-layers",
         type=int,
         default=0,
-        help="Freeze the first ``n`` layers of the FFN from the checkpoint model (specified by ``model-frzn``).",
+        help="Freeze the first ``n`` layers of the FFN from the checkpoint model (specified by ``checkpoint`` with `freeze-encoder`).",
```

Comment:
Should we add a check to require that `--checkpoint` is not None if `--freeze-encoder` is specified? Should we add a check to require that `--checkpoint` and `--freeze-encoder` are not None if `--frzn-ffn-layers` is specified?